deploy/backplane/cssre/30-cssre-mustgather.RoleBinding.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: backplane-cssre-mustgather
+  namespace: openshift-must-gather-operator
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:serviceaccounts:openshift-backplane-cee
```

Comment:
Should this be `openshift-backplane-cssre` ?

Reply:
Thank you @mrbarge, that was a mistake on my part. I have updated the code.